### PR TITLE
#8259

### PR DESCRIPTION
**Pull request by Usnul**
see #8259

### CODE REVIEW

**Comment by mrdoob** [comment]
Hmm, that code path is deprecated... How are you generating files that has `standard` as material?

**Comment by Usnul** [comment]
using the blender exporter (with manual editing to set material).

**Comment by mrdoob** [comment]
Uhm... We should update the exporter so it updates in the new format...

**Comment by Usnul** [comment]
so.. can we have this until an alternative workable path exists?

File: src/loaders/Loader.js [hunk 1]
```diff
@@ -121,6 +121,9 @@ THREE.Loader.prototype = {
 			};
 
 			for ( var name in m ) {
+				if ( !m.hasOwnProperty( name ) ) {
```
Comment (mrdoob) [inline src/loaders/Loader.js]: What's this for?

**Comment by Usnul** [comment]
this avoid inherited properties and is suggested by a few linters, i can't find a conclusive answer for this particular case other than - my lint told me to :)
Want it removed?

File: src/loaders/Loader.js (hunk 1 above)
Comment (Usnul) [inline src/loaders/Loader.js]: this avoid inherited properties and is suggested by a few linters, i can't find a conclusive answer for this particular case other than - my lint told me to :)
Want it removed?

Comment (mrdoob) [inline src/loaders/Loader.js]: Yes please :)

**Comment by Usnul** [comment]
done

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**